src/screens/Profile.js

```diff
@@ -35,6 +37,7 @@ function Profile() {
   const searchParams = useSearchParams()
   const selectedAccount = searchParams.get('account') || connectedAccount
   const accountStakes = useAccountStakes(selectedAccount)
+  const accountInactiveStakes = useInactiveProposalsWithStake()
```

Comment:
Would it make sense to use it inside the `InactiveProposalsStake` component directly?

Reply:
hmm not sure we do the same with the `stakingTokens` component, probably i would let like this for consistency